src/App.jsx

```diff
@@ -0,0 +1,62 @@
+import { useState } from 'react';
+import Container from 'react-bootstrap/Container';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+import Button from 'react-bootstrap/Button';
+import Form from 'react-bootstrap/Form';
+import InputGroup from 'react-bootstrap/InputGroup';
+
+import { useDispatch } from 'react-redux';
+import WeatherTable from './components/WeatherTable';
+import { searchForCity } from './reducers/weatherSlice';
+
+function App() {
+  const [search, setSearch] = useState('');
+  const dispatch = useDispatch();
+
+  const handleInput = (event) => {
+    setSearch(event.target.value);
+  };
+
+  const handleSearchButton = () => {
+    if (search !== '') {
```

Comment:
this could be simplified to if (search) - review the concept of truthy and falsy